app/javascript/shared/components/tags.jsx

```diff
@@ -356,7 +370,8 @@ class Tags extends Component {
     if (query === '') {
       return new Promise(resolve => {
         setTimeout(() => {
-          'search query'
+          'search query';
```

Comment:
I don't understand this line 🧐

Reply:
Hmm, indeed. That's interesting definitely but it wasn't me. There were a few conflicts and I just merged master and it's coming from there.